Update remote MCP OAuth availability (#9549)
fishtown-analytics/docs.getdbt.com: website/snippets/_mcp-qs-platform.md

For local CLI only (with or without a <Constant name="dbt_platform"/> account), see [Run dbt locally](/docs/dbt-ai/mcp-quickstart-cli) or [Run dbt Wizard locally](/docs/dbt-ai/wizard-quickstart).

 To configure or disable specific tools, see the [Environment variables reference](/docs/dbt-ai/mcp-environment-variables). 

## Prerequisites

- [Install uv](https://docs.astral.sh/uv/getting-started/installation/)
- A [<Constant name="dbt_platform"/> account](https://www.getdbt.com/signup)
- For OAuth connections:

>   - OAuth for the remote MCP is available for Starter, Enterprise, and Enterprise+ plans.

Reviewer: Hiya Mirna, this snippet appears on [Connect dbt MCP server to dbt platform
](https://docs.getdbt.com/docs/dbt-ai/mcp-quickstart-oauth?version=2.0&name=Fusion) which is about local MCP. It may confuse readers if there's reference to remote MCP in the prerequisites.

Author: same as earlier - you can use OAuth with the local dbt MCP server.

>   - OAuth for the remote MCP is available for Starter, Enterprise, and Enterprise+ plans.

Author: ```suggestion
  - MCP OAuth is available for Starter, Enterprise, and Enterprise+ plans.
```

  - An account admin has to enable AI features on your <Constant name="dbt_platform"/> account. Refer to [Enable AI features](/docs/platform/enable-dbt-ai) for more info.

## Step 1: Choose your auth method and configure

<Tabs groupId="auth">

<TabItem value="oauth" label="OAuth">

> _Remote MCP OAuth is available in public beta for Starter, Enterprise, and Enterprise+ accounts._

Reviewer: Similar to this [comment](https://github.com/dbt-labs/docs.getdbt.com/pull/9549#discussion_r3519236165).

Author: you can use OAuth with the local dbt MCP server.

Reviewer: Thanks Mirna — that makes sense, and I agree OAuth works with the local MCP server too.

The wording in `_mcp-qs-platform.md` still says "remote MCP" in the prerequisites and OAuth tab though, so it might confuse readers on the local quickstart page. Maybe something like:

Prerequisites: "OAuth for MCP is available for Starter, Enterprise, and Enterprise+ plans" (drop "remote")
OAuth tab: same — reference local MCP OAuth, not remote`

Also the line below still says "Enterprise and Enterprise+ accounts" only — should that include Starter to match the rest of the PR?

> _Remote MCP OAuth is available in public beta for Starter, Enterprise, and Enterprise+ accounts._

Author: ```suggestion
_MCP OAuth is available in public beta for Starter, Enterprise, and Enterprise+ accounts._
```

> OAuth is the fastest setup for <Constant name="dbt_platform"/> , no tokens to copy or manage. A browser window opens to authenticate the first time you connect. 

Reviewer: ```suggestion
OAuth is the fastest setup for <Constant name="dbt_platform"/> Enterprise and Enterprise+ accounts, no tokens to copy or manage. A browser window opens to authenticate the first time you connect. 
```

> OAuth is the fastest setup for <Constant name="dbt_platform"/> Enterprise and Enterprise+ accounts, no tokens to copy or manage. A browser window opens to authenticate the first time you connect. 

Author: ```suggestion
OAuth is the fastest setup for <Constant name="dbt_platform"/> accounts, no tokens to copy or manage. A browser window opens to authenticate the first time you connect. 
```

For local CLI only (with or without a <Constant name="dbt_platform"/> account), see [Run dbt locally](/docs/dbt-ai/mcp-quickstart-cli) or [Run dbt Wizard locally](/docs/dbt-ai/wizard-quickstart).

>  To configure or disable specific tools, see the [Environment variables reference](/docs/dbt-ai/mcp-environment-variables). Choose _OAuth_ (available for Starter, Enterprise, and Enterprise+ accounts) or _Tokens_ (more control, better for shared setups).

Author: ```suggestion
 To configure or disable specific tools, see the [Environment variables reference](/docs/dbt-ai/mcp-environment-variables). 
```

<TabItem value="oauth" label="OAuth">

> OAuth is the fastest setup for <Constant name="dbt_platform"/> Starter, Enterprise, and Enterprise+ accounts &mdash; no tokens to copy or manage. A browser window opens to authenticate the first time you connect. 

Author: ```suggestion
_Remote MCP OAuth is available in public beta for Starter, Enterprise, and Enterprise+ accounts._

OAuth is the fastest setup for <Constant name="dbt_platform"/> , no tokens to copy or manage. A browser window opens to authenticate the first time you connect. 
```

> For OAuth _without_ a local install, use the [remote MCP server](/docs/dbt-ai/mcp-quickstart-remote), available in public beta for Starter, Enterprise, and Enterprise+ accounts. If your client does not support OAuth or you need token-based access, use [token-based authentication](/docs/dbt-ai/setup-remote-mcp#token-based-authentication).

Author: ```suggestion
For OAuth _without_ a local install, use the [remote MCP server](/docs/dbt-ai/mcp-quickstart-remote). If your client does not support OAuth or you need token-based access, use [token-based authentication](/docs/dbt-ai/setup-remote-mcp#token-based-authentication).
```
